release-guide.md

```diff
@@ -0,0 +1,104 @@
+# SIARD Suite Release Testing Guide
+
+## Motivation
+
+The development of SIARD Suite is financed by the Swiss Federal Archives (SFA), which is one of the main stakeholders of the project. The SFA is also one of the main users of the application. There are many restrictions on the workstations that can be used by SFA staff:
```

Comment:
The SFA **are** also one of the main users of the application. There are many restrictions on **SFA staff workstations:**